packages/client/src/scripts/get-note-menu.ts

```diff
@@ -178,13 +183,16 @@ export function getNoteMenu(props: {
 			text: i18n.ts.copyLink,
 			action: copyLink
 		}, (appearNote.url || appearNote.uri) ? {
+			icon: 'fas fa-link',
+			text: i18n.ts.copyRemote,
+			action: copyRemote
+		} : undefined, (appearNote.url || appearNote.uri) ? {
```

Comment:
```suggestion
		}, {
```
`showOnRemote`と同じ条件式を使うので、そのまま下の条件式の中に項目だけ追加すればいいかなと思う
そのためここの判定はカット